fastparquet/parquet.thrift

```diff
@@ -312,6 +312,10 @@ enum Encoding {
   /** Dictionary encoding: the ids are encoded using the RLE encoding
    */
   RLE_DICTIONARY = 8;
+
+  /** Custom lossy rice encoding
+   */
+  LOSSY_RICE = 19850410;
```

Comment:
I of course understand the portability concerns. There are real tradeoffs to be had here though.